text/rocq-roadmap.md

```diff
@@ -10,7 +10,8 @@ While Rocq leverages the rich heritage of Coq, it embraces a more agile developm
 This roadmap outlines our vision for Rocq's future, focusing on four key pillars:
 
 + **Open and Accessible**: The Rocq Prover provides user-friendly installation, comprehensive educational resources, and a thriving community to ensure everyone can benefit from its power.
-+ **Trustworthy**: The Rocq Prover prioritizes self-verification and performance optimization, ensuring the highest level of trust and efficiency in your formal proofs.
++ **Trustworthy**: The Rocq Prover prioritizes self-verification, ensuring the highest level of trust in your formal proofs.
++ **Performance**: We are committed to optimizing the system for our flagship projects in both industry and academia.
```

Comment:
Repetition of "We are committed to" on two subsequent lines (this one and the next that you did not change).

Furthermore, it should be "Performant" rather than "Performance" for consistency with the surrounding bullet points.

What about:

```suggestion
+ **Performant**: We optimize the system for our flagship projects in both industry and academia.
```